### DISCUSSION TITLE
Black Screen

### DISCUSSION THREAD

**Discussion by Maxiroon** [project64/project64#2185] [Support]
I just got Project64 and the Screen is black, no matter what game it is. Sometimes the HUD appears, but not the game itself. The audio is also just fine. How do I fix this?
![Black](https://user-images.githubusercontent.com/76464374/151715253-30583eed-e6a1-4346-a850-c850c61ddbe5.png)

**Comment by DerekTurtleRoe** [comment]
@QwertyNavy We are going to need a lot more information to provide assistance. Please provide your PC specs, OS version, Project64 version, plugins used, and the games affected.

**Comment by ObiW4nKenobi** [comment]
CPU - Ryzen 5 5600G
Motherboard - Gigabyte A520M DS3H AC
Memory Kit - 16GB (2x8GB G.Skill FlareX 3200MHz)
Windows 11 25H2
Project-64 Build 6120 and 6179
Plugins used: Only what comes with the zipped file
Games affected: ALL, I couldn't get any of my ROMs to work with this. Only a black screen after selecting multiple different ROMs.
